Source/Scene/Points3DTileContent.js

```diff
@@ -166,11 +169,46 @@ define([
         var pointsLength = view.getUint32(byteOffset, true);
         byteOffset += sizeOfUint32;
 
-        var positionsOffsetInBytes = byteOffset;
-        var positions = new Float32Array(arrayBuffer, positionsOffsetInBytes, pointsLength * 3);
+        var batchTableJSONByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var batchTableBinaryByteLength = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUint32;
+
+        var positions = new Float32Array(arrayBuffer, byteOffset, pointsLength * 3);
+        byteOffset += pointsLength * 3 * sizeOfFloat32;
 
-        var colorsOffsetInBytes = positionsOffsetInBytes + (pointsLength * (3 * Float32Array.BYTES_PER_ELEMENT));
-        var colors = new Uint8Array(arrayBuffer, colorsOffsetInBytes, pointsLength * 3);
+        var colors;
+        var translucent = false;
+
+        if (batchTableJSONByteLength > 0) {
+            // Get the batch table JSON
+            var batchTableString = getStringFromTypedArray(uint8Array, byteOffset, batchTableJSONByteLength);
+            var batchTableJSON = JSON.parse(batchTableString);
+            byteOffset += batchTableJSONByteLength;
+
+            // Get the batch table binary
+            var batchTableBinary;
+            if (batchTableBinaryByteLength > 0) {
+                batchTableBinary = new Uint8Array(arrayBuffer, byteOffset, batchTableBinaryByteLength);
+            }
+            byteOffset += batchTableBinaryByteLength;
+
+            // Get the point colors
+            var tiles3DRGB = batchTableJSON.TILES3D_RGB;
+            var tiles3DRGBA = batchTableJSON.TILES3D_RGBA;
+            if (defined(tiles3DRGB)) {
```

Comment:
What do you think about https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/22#issuecomment-232489831

Should the tile be invalid (in practice if someone wants multiple per-point colors, they would use their own semantics)?  Or give should RGBA win over RGB?